opensearch/aryn-opensearch.sh

```diff
@@ -0,0 +1,427 @@
+#!/bin/bash
+# TODO: eric - detect low disk space and error out.
```

Comment:
remove 'eric'.  do you want to open an issue and assign it to yourself?

Reply:
The eric was the reference for the bug.
Replaced with TODO: http://...

will do all the other ones also.